hack/lint-addons/lint-addons.go

```diff
@@ -11,6 +11,7 @@ import (
 	"strings"
 
 	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
```

Comment:
can this be reverted from this PR? I'd like to revisit if/how we should use pkg/errors separately as a team